packages/types/lib/auth/http.api.ts

```diff
@@ -109,16 +109,7 @@ export type PostPublicTableauAuthorization = Endpoint<{
 
 export type PostPublicJwtAuthorization = Endpoint<{
     Method: 'POST';
-    Body: {
-        privateKeyId?: string;
-        issuerId?: string;
-        privateKey:
-            | {
-                  id: string;
-                  secret: string;
-              }
-            | string;
-    };
+    Body: Record<string, any>;
```

Comment:
Same here, there is some inconsistencies with the type. Could at least be like this no?
```ts
{ 
[key: string]: string;
privateKey?: ...
}
```